dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -1289,7 +1331,7 @@ private Role resolveRole(final String id) throws DotDataException {
 
         Role role = null;
         final String newid = id.substring
-                (id.indexOf("-") + 1, id.length());
+                (id.indexOf("-") + 1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [String.indexOf(char) is faster than String.indexOf(String).](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4801230)